test/util/model-tests.js

```diff
@@ -235,6 +235,27 @@ module.exports = function defineModelTestsWithDataSource(options) {
       });
     });
 
+    describe('Model.exists(id, [callback])', function() {
+      it('returns true when the model with the given id exists', function(done) {
+        User.create({first: 'max'}, function(err, user) {
+          if (err) return done(err);
+          User.exists(user.id, function(err, exist) {
+            if (err) return done(err);
+            assert.equal(exist, true);
+            done();
+          });
+        });
+      });
+
+      it('returns false when there is no model with the given id', function(done) {
```

Comment:
I have also added a test to verify what happens when the target model does not exist - we want to receive successful response with `false` value, not a `404 Not Found` error.